### ISSUE TITLE
Mobile A11y Testing 11/1

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-styles#64]
This is mostly totally fine. Just note that the red plus button isn't implemented as a button -- it's missing that role. I know this isn't crucial for this demo though, just wanted to flag.

**Comment by notwaldorf** [comment]
@lpalmaro The red plus button isn't a button -- it's a plain div, that isn't clickable, and just looks like a fab button to demonstrate layouts. I don't think it's worth implementing it as a button -- the whole point of the demo is to layout divs, not elements. Nothing is really tabbable or interact-able either, I don't think.

**Comment by lpalmaro** [comment]
SG - thanks for the clarification!
